wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/UpNextScreen.kt

```diff
@@ -24,7 +24,7 @@ object UpNextScreen {
 
 @Composable
 fun UpNextScreen(
-    onNavigateToNowPlaying: (String) -> Unit,
+    navigateToNowPlaying: () -> Unit,
```

Comment:
Since this function is actually doing the navigating, I thought that dropping the "on" made sense.